source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -535,11 +535,11 @@ bool ListParser::MatchNewOrderedListItem(std::stringstream &stream, std::string
 // we do not include in the current block, we return, and have it handled by the caller
 void ListParser::ParseSubBlocks(std::stringstream &stream)
 {
-    while (!stream.eof()) 
-    { 
+    while (!stream.eof())
+    {
         if (IsNewLine(stream.peek()))
         {
-            int newLineChar = stream.get();
+            int newLineChar = static_cast<char>(stream.get());
```

Comment:
wrong cast.

Reply:
Hmm, os repo was complaining about a narrowing cast here.

Reply:
what are you doing here. i thought you left... :-) that's weird. get returns int. may be windows version returns long?

---
In reply to: [163430153](https://github.com/Microsoft/AdaptiveCards/pull/1091#discussion_r163430153) [](ancestors = 163430153)

Reply:
You were right (as expected), I changed this one without any need for it. I'll revert it